packages/azure_frontdoor/_dev/build/build.yml

```
@@ -0,0 +1,3 @@
+dependencies:
+  ecs:
+    reference: git@1.12
```

```suggestion
    reference: git@v8.5.0
```
Needs to be 8.5 and above, you can use  8.5.1  also but it requires to be changed throughout accordingly.

---

Hello @ShourieG , I got an error with 8.5 so I used 8.4 instead of that. Is this okay?

---

what exact errors are you facing ?

---

With 8.5, I got this.
```
Error: updating files failed: updating readme file README.md failed: rendering Readme failed: executing template failed: template: README.md:21:2: executing "README.md" at <fields "deep_security">: error calling fields: can't create fields validator instance (path: /mnt/c/k/current/trendmicro/integrations/packages/trendmicro/data_stream/deep_security): can't create field dependency manager: can't build fields schema: can't load fields: unmarshalling field body failed: yaml: unmarshal errors:Error: updating files failed: updating readme file README.md failed: rendering Readme failed: executing template failed: template: README.md:21:2: executing "README.md" at <fields "deep_security">: error calling fields: can't create fields validator instance (path: /mnt/c/k/current/trendmicro/integrations/packages/trendmicro/data_stream/deep_security): can't create field dependency manager: can't build fields schema: can't load fields: unmarshalling field body failed: yaml: unmarshal errors:
```